lib/jekyll/utils.rb

```diff
@@ -194,26 +200,10 @@ def slugify(string, mode: nil, cased: false)
         return cased ? string : string.downcase
       end
 
-      # Replace each character sequence with a hyphen
-      re =
-        case mode
-        when "raw"
-          SLUGIFY_RAW_REGEXP
-        when "default"
-          SLUGIFY_DEFAULT_REGEXP
-        when "pretty"
-          # "._~!$&'()+,;=@" is human readable (not URI-escaped) in URL
-          # and is allowed in both extN and NTFS.
-          SLUGIFY_PRETTY_REGEXP
-        when "ascii"
-          # For web servers not being able to handle Unicode, the safe
-          # method is to ditch anything else but latin letters and numeric
-          # digits.
-          SLUGIFY_ASCII_REGEXP
-        end
+      # Drop accent marks from latin characters. Everything else turns to ?
+      string = ::I18n.transliterate(string) if mode == "latin"
 
-      # Strip according to the mode
-      slug = string.gsub(re, "-")
+      slug = replace_character_sequence_with_hyphen(string, :mode => mode)
```

Comment:
Using the symbol syntax would be better here, so that you're consistent with the rest of the codebase.
```ruby
mode: mode
```

Reply:
@mattr- Another style error in the build came up here as well, something about using hash rocket syntax, so I wrote it this way.